src/Sentry.Profiling/SamplingTransactionProfiler.cs

```diff
@@ -80,6 +80,7 @@ private void OnThreadSample(TraceEvent data)
             }
             else
             {
+                _options.LogDebug("Profiler received a sample past the end time; stopping.");
```

Comment:
Is thsi something we should look into?
If so, could be a warn?

```suggestion
                _options.LogWarning("Profiler received a sample past the end time; stopping.");
```

Reply:
it's not unexpected, I'll reword